documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -163,6 +163,8 @@ Reusing components like this means that things like sidebar scroll state are pre
 {/key}
 ```
 
+> Instead of `onMount` and `onDestroy` you can use [aferNavigate](modules#$app-navigation-afternavigate) and [beforeNaviate](modules#$app-navigation-beforenavigate) respectively.
```

Comment:
I think it would be better to have this up there after the "no will lifecycle methods run" sentence.

Reply:
Yeah, I wasn't sure where to put it to not break the 'flow' of going into the reactivity example, should it then be before the code sample or after ?

Reply:
Moved it